### ISSUE TITLE
Trying SSH access

### ISSUE THREAD

**Issue by coniman** [MTJailed/jailbreakme#10]
Hi, I've been testing your jailbreakme solution on an iPhone 8. Everything works fine, empty_list payload works fine and loads (seen on the console of Xcode connecting over USB). I'm trying to connect to my iPhone via SSH but I always get the refused connection message. I don't know if it's because of the limitations of the exploit or if I'm doing something wrong.

Also I'm wondering if it's possible to create a WebKit exploit based on Electra 11.3.1 Jailbreak now that is out.

Thanks in advance.

**Comment by MTJailed** [comment]
Yes it is, but it seems someone more skilled is working on it and I will wait until he / she finishes it then add his into my unified jailbreakme, why? because I care about stability!

**Comment by MTJailed** [comment]
You could help by providing the emptylist log as then i know what goes wrong

**Comment by MTJailed** [comment]
Did you prepare the directory by uploading the required files firstly?

**Comment by coniman** [comment]
I don't know how to upload the required files. I run an Apache server on my Mac serving this folder. I'm also a little bit lost, in the old times I used to jailbreak my devices over USB using tools like redsn0w an evasi0n, so this is new to me.  I'm really interested in this project because I got scammed on eBay with an iCloud locked iPhone 8, so jailbreaking it is the only way to bypass the lock. I'm accessing to your tool through the captive portal modifying DNS on the device.

**Comment by jujijoog** [comment]
Hey @coniman I’m about 95 percent sure that you cannot jailbreak while using captive portal on a cloud locked phone. This is very interesting to me though. Did the exploit say that they ran successfully? What is your plan to delete setup.app once you gain ssh root? If you were able to gain read/write on root that would work, however I highly doubt that will happen.

**Comment by coniman** [comment]
After a few tries and reboots the exploit seems to work but no SSH, this is what I see in the Xcode Console 
[Untitled.pdf](https://github.com/MTJailed/jailbreakme/files/2173309/Untitled.pdf)

**Comment by coniman** [comment]
It seems to work, I get a blank page after the exploit load. It's that what I should get? Yes, my plan is to delete Setup.app to reach the SpringBoard and use it as an iPod Touch at least.

Also, I want to thank you for your interest .

**Comment by jujijoog** [comment]
HOLY SHIT. It looks like empty_list ran successfully! was not expecting that. however it didnt even try to initialize ssh. but im not an expert, not by a long shot lets see what mtjailed has to say about those logs but dont get your hopes up.

**Comment by jujijoog** [comment]
can you post a screenshot of the white screen you are talking about. was it outside of the captive portal window?

**Comment by jujijoog** [comment]
im gonna guess that qlin loaded but was not able to do anything but thats a guess

**Comment by jujijoog** [comment]
another thing is you are on 11.3.1 and i dont think jailbreak has r/w remount for 11.3.1 yet

**Comment by jujijoog** [comment]
also im fairly sure setup.app is running while you are in captive portal. it wont let you delete it if its running. i think you would have to be in dfu mode or already bypassed to do this. however if you do get root somehow there are a lot of methods you could try to bypass

**Comment by coniman** [comment]
Sometimes the exploits fail, like in this video https://drive.google.com/open?id=1Pgra9YTEfE5CD_Mk3rAETsDZcmtGBYhe , I think it's because the DNS server has ads and they load every time changing the memory address.

Here it's the white screen that I get when the exploit loads: https://drive.google.com/open?id=1P5dRAitjtdppr5YidgHoHjvnTClZY0lK

**Comment by jujijoog** [comment]
no its normal for the exploit to fail. empty_list is unstable

**Comment by jujijoog** [comment]
i explained what are you trying to do to some people in my telegram group and

**Comment by coniman** [comment]
Well, maybe not delete the app but kill the process might be easy.

**Comment by jujijoog** [comment]
omg it worked hahaha.  i

**Comment by jujijoog** [comment]
i just opened filza and revoked read write access to filza.app while running it and it allowed me. then i exited and tried to load filza and it crashes now and wont open

**Comment by jujijoog** [comment]
so yeah chmod 0600 would work to bypass setup.app i think if you have r/w remount and root. you would just need to reboot phone. however getting root and remount is another story

**Comment by jujijoog** [comment]
so for r/w you will have to wait im pretty sure. what you could do is try just ending the setup.app proc

**Comment by jujijoog** [comment]
but to be honest if you get root through captive portal within setup.app i will be very very surprised and also you would be the first person ever to do that to my knowledge and if i were you i would not tell anybody and report it to apple for a bunch of cash and then just by a new phone lmao

**Comment by coniman** [comment]
Hahahaha that would be great! :P. So now the question is how to get SSH access to try that.

**Comment by jujijoog** [comment]
lol i doubt they would pay you. they would probably say its ians and just  your applied method and then  patch their secuirty in setup.app for ios 12

**Comment by jujijoog** [comment]
i think your best shot would be to try using standard methods like emoji pasting and other buffer overload techniques while running empty_list at the same time in the hopes that you crash to springboard.

**Comment by jujijoog** [comment]
i was actually able to crash setup.app to springboard for a couple seconds although it was frozen/ unresponsive. i used a combination of emoji exponentiation overflow and spamming call and end call on the emergency call menu and random manic swiping with voice over on. if you were to do all that and manage to start running the empty_list exploit it would really fuck shit up and could crash setup,app. or it could just force reboot or something but thats what i would try

**Comment by jujijoog** [comment]
you have to dial a legit emergency number like 112 though lol and if you time it wrong the emercncy responder answers

**Comment by coniman** [comment]
I’ve tried every emoji in the world and it’s imposible to crash setup.app. empty_list starts running as com.apple.WebKit and Setup.app as com.apple.setupd (I think), so they are separated processes. When empty_list crashes the hole device reboots itself and not only the app. I don’t know if it’s possible to just crash the app.

**Comment by coniman** [comment]
I think they’ve patched every emoji crash in 11.3.1 cause I,ve tried everything

**Comment by MTJailed** [comment]
3:26.369381 +0200 com.apple.WebKit.WebContent XXX Qilin initialized
05:23:26.369546 +0200 com.apple.WebKit.WebContent XXX User ID now: 0 0

@coniman You got as far as letting QiLin gain root :)

I'll add the binary pack soon to the release tap and add documentation on how to prepare it in DOCS

**Comment by coniman** [comment]
Great!

Thanks

**Comment by MTJailed** [comment]
UFOFinder 2 has now been release with arm64 iOS 10 - iOS 11.4.1 support!
It's way more stable and can find the link code gadget as wel

**Comment by coniman** [comment]
Where I can find bootstrap.zip?

**Comment by coniman** [comment]
I read the documentation but I can't find the file

**Comment by MTJailed** [comment]
Its because I didn’t add it yet, i’ll do that today ;)

**Comment by coniman** [comment]
Ok ;)

**Comment by coniman** [comment]
Any progress?

**Comment by DevX-Cipher** [comment]
You know bypassing and deleting tfp0 aka Lucas exploit I already have something stable enough to tfp0 on Dsn portal stuff and atm I'm trying to delete the setup.app on 32bit devices and I need a tester but yer I can't do much testing I don't have those devices 

Even if you get tfp0 I was hoping to right another exploit to be able show root file through the browser even tho it's a portal that what exploits are for but if people wanna help that would be great

Also the only reason no ssh because it's not installed on your device

**Comment by coniman** [comment]
I don’t have any 32bit device. But if you have something to delete Setup.app in 64bit please share it ;)
